test/integration/features/generic-driver.feature

```diff
@@ -0,0 +1,49 @@
+@generic-driver
+Feature: With generic driver Minishift can provision remote unprovisioned VM 
+
+  Scenario: User creates an unprovisioned VM
+    Given setting up environment variable "MINISHIFT_ENABLE_EXPERIMENTAL" with value "y" succeeds
+    When executing "minishift profile set remote" succeeds
+    And executing "minishift start --no-provision" succeeds
+    Then Minishift should have state "Running"
+
+  Scenario: Provisioning remote instance
+    Given unset environment variable "MINISHIFT_ENABLE_EXPERIMENTAL" succeeds
```

Comment:
The alignment is right justified not left justified :) @agajdosi  See below

![screenshot from 2018-07-04 18-42-49](https://user-images.githubusercontent.com/1132451/42279060-2d82bfbc-7fba-11e8-82ad-988d5939b8d9.png)